package.json

```diff
@@ -77,7 +77,8 @@
     "*.{{,c,m}j,t}s{,x}": [
       "eslint --ignore-pattern '!*'",
       "jest --testRegex='^(?!.*.integ.spec.js$).*.spec.js$' --findRelatedTests --passWithNoTests"
-    ]
+    ],
+    "@xen-orchestra/lite/**/*.{ts,vue}": "vue-tsc --noEmit"
```

Comment:
This will not work.

This configuration means "execute `vue-tsc --noEmit <filename>` for each modified file.

Except that `tsc` (and therefore `vue-tsc`) doesn't accept a file in parameter (well, more exactly, it accepts a file but then it ignores the `tsconfig.json` file so it fails)

Reply:
In that case maybe it should simply be put in the `test` script as I expect a `*.{ts,vue}` file to almost always be edited when making change in this package?

Reply:
Yes, we probably should do that for now.